docs/en_US/Compression/compression_pipeline_example.ipynb

```diff
@@ -0,0 +1,1306 @@
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "73e4227a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Train Epoch: 0 [0/60000 (0%)]\tLoss: 0.349769\n",
+      "Train Epoch: 0 [6400/60000 (11%)]\tLoss: 0.042060\n",
+      "Train Epoch: 0 [12800/60000 (21%)]\tLoss: 0.052566\n",
+      "Train Epoch: 0 [19200/60000 (32%)]\tLoss: 0.011052\n",
+      "Train Epoch: 0 [25600/60000 (43%)]\tLoss: 0.006365\n",
+      "Train Epoch: 0 [32000/60000 (53%)]\tLoss: 0.008099\n",
+      "Train Epoch: 0 [38400/60000 (64%)]\tLoss: 0.024377\n",
+      "Train Epoch: 0 [44800/60000 (75%)]\tLoss: 0.026260\n",
+      "Train Epoch: 0 [51200/60000 (85%)]\tLoss: 0.038423\n",
+      "Train Epoch: 0 [57600/60000 (96%)]\tLoss: 0.014146\n",
+      "\n",
+      "Test set: Average loss: 0.0244, Accuracy: 9914/10000 (99%)\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# finetune the model to recover the accuracy.\n",
+    "\n",
+    "optimizer = torch.optim.SGD(model.parameters(), lr=0.01)\n",
+    "\n",
+    "for epoch in range(0, 1):\n",
+    "    trainer(model, optimizer, criterion, epoch)\n",
+    "    evaluator(model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea513df7",
+   "metadata": {},
+   "source": [
+    "# 5. Prepare config_list for quantization"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "dc0dde77",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config_list = [{\n",
+    "    'quant_types': ['weight'],\n",
+    "    'quant_bits': {'weight': 8},\n",
+    "    'op_names': ['conv1', 'conv2']\n",
+    "}]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48f4aba2",
+   "metadata": {},
+   "source": [
+    "# 6. Choose a quantizer and quantizing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "43d9deb4",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "NaiveModel(\n",
+       "  (conv1): QuantizerModuleWrapper(\n",
+       "    (module): Conv2d(1, 10, kernel_size=(5, 5), stride=(1, 1))\n",
+       "  )\n",
+       "  (conv2): QuantizerModuleWrapper(\n",
+       "    (module): Conv2d(10, 50, kernel_size=(5, 5), stride=(1, 1))\n",
+       "  )\n",
+       "  (fc1): Linear(in_features=800, out_features=500, bias=True)\n",
+       "  (fc2): Linear(in_features=500, out_features=10, bias=True)\n",
+       "  (relu1): ReLU6()\n",
+       "  (relu2): ReLU6()\n",
+       "  (relu3): ReLU6()\n",
+       "  (max_pool1): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)\n",
+       "  (max_pool2): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)\n",
+       ")"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from nni.algorithms.compression.pytorch.quantization import DoReFaQuantizer\n",
```

Comment:
Recommend using `QAT_Quantizer` instead of `DoReFaQuantizer` since quantization speedup only support `QAT_Quantizer` right now.

Reply:
got it, change to `QAT_Quantizer`